src/components/label/LabelPrimitive.js

```diff
@@ -5,12 +5,12 @@ import classNames from 'classnames';
 type Props = {
     children: React.Node,
     className?: string,
+    htmlFor?: string,
     labelContent: React.Node,
 };
 
-const LabelPrimitive = ({ children, className, labelContent, ...rest }: Props) => (
-    // eslint-disable-next-line jsx-a11y/label-has-for
-    <label>
+const LabelPrimitive = ({ children, className, labelContent, htmlFor, ...rest }: Props) => (
+    <label htmlFor={htmlFor}>
```

Comment:
I don't think `htmlFor` is necessary when the content is nested inside the `label` tag. The browser should associate them automatically.

Reply:
We should support using it unnested, though, right? Since the fact that this is a plain `label` without classname is not guaranteed.

Reply:
because we decided to go with making it all have to be nested anyways, I'm just gonna remove it and resolve this.